src/firefly/main_window.py

```diff
@@ -10,6 +10,8 @@
 
 from haven import load_config
 
+from .queue_client import is_in_use
```

Comment:
Seems strange this package starts with a "."

Reply:
That's a relative import. It's equivalent to `from firefly.queue_client import is_in_use`.

PEP8 recommends the absolute import. I'll update it. https://peps.python.org/pep-0008/